tests/unit/algo/classification/heads/test_custom_hlabel_cls_head.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import pytest
+import torch
+from mmpretrain.structures import DataSample
+from otx.algo.classification.heads.custom_hlabel_cls_head import CustomHierarchicalClsHead
+from otx.core.data.entity.classification import HLabelInfo
+from torch import nn
+
+
+class TestCustomHierarchicalClsHead:
+    @pytest.fixture()
+    def fxt_hlabel_head(self) -> nn.Module:
+        return CustomHierarchicalClsHead(
+            num_multiclass_heads=3,
+            num_multilabel_classes=0,
+            in_channels=24,
+            num_classes=6,
+            multiclass_loss_cfg={
+                "type": "CrossEntropyLoss",
+                "use_sigmoid": False,
+                "reduction": "mean",
+                "loss_weight": 1.0,
+            },
+        )
+
+    @pytest.fixture()
+    def fxt_data_sample(self) -> DataSample:
+        data_sample = DataSample(
+            img_shape=(24, 24, 3),
+            gt_label=torch.zeros(6, dtype=torch.long),
+            hlabel_info=HLabelInfo(
+                num_multiclass_heads=3,
+                num_multilabel_classes=0,
+                head_idx_to_logits_range={"0": (0, 2), "1": (2, 4), "2": (4, 6)},
+                num_single_label_classes=6,
+                empty_multiclass_head_indices=[],
+                class_to_group_idx={
+                    "0": (0, 0),
+                    "1": (0, 1),
+                    "2": (1, 0),
+                    "3": (1, 1),
+                    "4": (2, 0),
+                    "5": (2, 1),
+                },
+                all_groups=[["0", "1"], ["2", "3"], ["4", "5"]],
+                label_to_idx={
+                    "0": 0,
+                    "1": 1,
+                    "2": 2,
+                    "3": 3,
+                    "4": 4,
+                    "5": 5,
+                },
+            ),
+        )
+        return [data_sample, data_sample]
```

Comment:
Would you move this to more common place so that can be used by other tests? For example,
https://github.com/openvinotoolkit/training_extensions/blob/3441111af702831c836d070ae5d62a4b7932209a/tests/conftest.py#L12-L23

Reply:
Done. https://github.com/openvinotoolkit/training_extensions/pull/2761/commits/fdc033ec6b1fdc6615705ddd75af960a49af9f1d